cluster/gce/util.sh

```diff
@@ -836,6 +851,12 @@ function delete-network() {
 
 function delete-subnetworks() {
   if [[ ${ENABLE_IP_ALIASES:-} != "true" ]]; then
+    echo "Deleting default subnets..."
+    # This value should be kept in sync with number of regions.
+    local parallelism=9
+    gcloud compute networks subnets list | \
+    grep "${NETWORK}" | tr -s " " | cut -f2 -d" " | \
```

Comment:
- Will this try to delete the auto subnetworks?
- Can we tighten the grep so a simple string like "e2e" will match just the e2e network, rather than substrings as well in other columns?
- Indent continuation